lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/TransactionSpec.hs

```diff
@@ -478,6 +506,27 @@ goldenTestStdTx tl keystore inps outs bytes' = it title $ do
   where
     title = "golden test mkStdTx: " <> show inps <> show outs
 
+goldenTestDelegationCertTx
+    :: forall t k. (t ~ Jormungandr, HasCallStack)
+    => TransactionLayer t k
+    -> (Address -> Maybe (k 'AddressK XPrv, Passphrase "encryption"))
+    -> PoolId
+    -> (k 'AddressK XPrv, Passphrase "encryption")
+    -> [(TxIn, TxOut)]
+    -> [TxOut]
+    -> ByteString
+    -> SpecWith ()
+goldenTestDelegationCertTx tl keystore pool (accountXPrv, pass) inps outs bytes' = it title $ do
+    let res = mkDelegationCertTx tl pool (accountXPrv, pass) keystore inps outs
+    let sealed = getSealedTx . snd <$> res
+    sealed `shouldBe` (Right $ unsafeFromHex bytes')
+    & counterexample ("poolId = " <> showHex (getPoolId pool))
+  where
+    title = "golden test mkCertificateTx: " <> show pool
+        <> show inps <> show outs
+    (&) = flip ($)
```

Comment:
**[change required]**

Use [(&) from `Data.Function` ](https://hackage.haskell.org/package/base-4.12.0.0/docs/Data-Function.html#v:-38-) 

**Why?**
Avoid superfluous definition from a function defined the `base` package.

Reply:
Done